SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -190,6 +191,63 @@ int ASpatialFunctionalTest::GetNumberOfClientWorkers()
 	return Counter;
 }
 
+void ASpatialFunctionalTest::AddActorDelegation_Implementation(AActor* Actor, uint8 ServerWorkerId, bool bPersistOnTestFinished /*= false*/)
+{
+	ISpatialFunctionalTestLBDelegationInterface* DelegationInterface = GetDelegationInterface();
+
+	if (DelegationInterface != nullptr)
+	{
+		bool bAddedDelegation = DelegationInterface->AddActorDelegation(Actor, ServerWorkerId, bPersistOnTestFinished);
+		ensureMsgf(bAddedDelegation, TEXT("Tried to delegate Actor %s to Server Worker %d but couldn't"), *Actor->GetName(), ServerWorkerId);
+	}
+}
+
+void ASpatialFunctionalTest::RemoveActorDelegation_Implementation(AActor* Actor)
+{
+	ISpatialFunctionalTestLBDelegationInterface* DelegationInterface = GetDelegationInterface();
+
+	if (DelegationInterface != nullptr)
+	{
+		bool bRemovedDelegation = DelegationInterface->RemoveActorDelegation(Actor);
+		ensureMsgf(bRemovedDelegation, TEXT("Tried to remove Delegation from Actor %s but couldn't"), *Actor->GetName());
```

Comment:
```suggestion
		ensureMsgf(bRemovedDelegation, TEXT("Tried to remove Delegation from Actor %s but couldn't"), *GetNameSafe(Actor));
```
Just to be super-safe for nullptr Actors.